glide-core/src/client/value_conversion.rs

```diff
@@ -175,6 +176,21 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        ExpectedReturnType::ArrayOfKeyValuePairs => match value {
+            Value::Nil => Ok(value.clone()),
+            Value::Array(ref array) if array.is_empty() || matches!(array[0], Value::Array(_)) => {
```

Comment:
when it could be empty?

Reply:
If the count value was supplied but the key doesn't exist